### ISSUE TITLE
DotNet Aspire

### ISSUE THREAD

**Issue by petrsvihlik** [petrsvihlik/WopiHost#98]
https://github.com/dotnet/tye
https://mybuild.microsoft.com/sessions/b7e27509-c56c-42ad-9ce2-34270ecb0a38?source=sessions

**Comment by petrsvihlik** [comment]
debugging: https://ml-software.ch/posts/first-look-at-project-tye

**Comment by petrsvihlik** [comment]
This has shifted towards https://learn.microsoft.com/en-us/dotnet/aspire/get-started/aspire-overview
Let's explore whether it could be useful for WOPI.

**Comment by dviry** [comment]
AFAIK this is for Distributed applications using micro-services, not sure how this applies to this project...

**Comment by petrsvihlik** [comment]
yeah, I used tye (experiment linked above) to setup the whole infrastructure and "tie" it together (all the hostnames, ports, etc.. so that the config was much easier)...not sure if they kept any aspects of this in aspier

**Comment by petrsvihlik** [comment]
could be used to start up the wopihost.web and wopihost and tie them together...
https://youtu.be/fiePiEc1qcU
